spec/support/corpus.rb

```diff
@@ -124,6 +116,52 @@ def checkout
 
     private
 
+      # Count mutations generated for provided source file
+      #
+      # @param path [Pathname] path to a source file
+      #
+      # @return [Fixnum] number of mutations generated
+      def count_mutations(path)
+        node = Parser::CurrentRuby.parse(path.read)
+
+        return DEFAULT_MUTATION_COUNT unless node
+
+        Mutant::Mutator::REGISTRY.call(node).length
+      end
+
+      # Yield and conditionally suppress errors according to whitelist
+      #
+      # @param path [Pathname] path responsible for exception
+      # @param default_return [Object] return value if exception is permitted
+      #
+      # @return value returned from yield if no exception raised
+      # @return `default_return` otherwise
+      def filter_exceptions(path, default_return)
```

Comment:
Mutant is in a hard domain (mostly because it works on ruby a known to be unsharp langauge), keeping the tool sharp for my use cases and the maintenance overhead low is the main priority.
I do not run any tool that requires legacy parser versions.